config/crd/bases/monitoring.gitpod.io_cells.yaml

```diff
@@ -622,27 +622,75 @@ spec:
               apiServerReady:
                 description: APIServerReady reports whether Prometheus is able to
                   scrape node-exporter metrics or not
-                type: boolean
+                properties:
```

Comment:
Would it be worth switching to a [YAML anchor tag](https://medium.com/@kinghuang/docker-compose-anchors-aliases-extensions-a1e4105d70bd) to avoid the repetition?